website/docs/r/eri_router_to_azure_microsoft_connection_v1.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: "fic"
+page_title: "Flexible InterConnect: fic_eri_router_to_azure_microsoft_connection_v1"
+sidebar_current: "docs-fic-resource-eri-router-to-azure-microsoft-connection-v1"
+description: |-
+  Manages a V1 Router to Azure Microsoft Connection resource within Flexible InterConnect.
+---
+
+# fic\_eri\_router\_to\_azure\_microsoft\_connection\_v1
+
+Manages a V1 Router to Azure Microsoft Connection resource within Flexible InterConnect.
+
+## Example Usage
+
+### Basic Usage
+
+```hcl
+resource "fic_eri_router_v1" "router_1" {
+    name = "terraform_router_1"
+    area = "%s"
+    user_ip_address = "10.0.0.0/27"
+    redundant = false
+}
+
+resource "fic_eri_nat_component_v1" "nat_1" {
+    router_id = "${fic_eri_router_v1.router_1.id}"
+    nat_id = "${fic_eri_router_v1.router_1.nat_id}"
+
+    user_ip_addresses = [
+        "192.168.0.0/30",
+        "192.168.0.4/30",
+        "192.168.0.8/30",
+        "192.168.0.12/30",
+    ]
+
+    global_ip_address_sets  {
+        name = "src-set-01"
+        type = "sourceNapt"
+        number_of_addresses = 5
+    }
+}
+
+resource "fic_eri_nat_global_ip_address_set_v1" "gip_1" {
+    depends_on = ["fic_eri_nat_component_v1.nat_1"]
+
+    router_id = "${fic_eri_router_v1.router_1.id}"
+    nat_id = "${fic_eri_router_v1.router_1.nat_id}"
+
+    name = "src-set-02"
+    type = "sourceNapt"
+    number_of_addresses = 5
+}
+
+resource "fic_eri_router_to_azure_microsoft_connection_v1" "connection_1" {
+    depends_on = ["fic_eri_nat_component_v1.nat_1"]
+
+    name = "terraform_connection_1"
+
+    source_router_id = "${fic_eri_router_v1.router_1.id}"
+    source_group_name = "group_1"
+    source_route_filter_in = "noRoute"
+    source_route_filter_out = "noRoute"
+
+    destination_interconnect = "Osaka-1"
+    destination_qos_type = "guarantee"
+    destination_service_key = "%s"
```

Comment:
"%s" is golang's format.
Please replace this into some string.

Reply:
Fixed!